api/tests/routes/native/v1/bookings_test.py

```diff
@@ -18,15 +18,12 @@
 from pcapi.core.categories import subcategories_v2 as subcategories
 from pcapi.core.external_bookings.factories import ExternalBookingFactory
 from pcapi.core.finance import utils as finance_utils
+from pcapi.core.geography.factories import AddressFactory
 import pcapi.core.mails.testing as mails_testing
+from pcapi.core.offerers import factories as offerer_factories
```

Comment:
@xordoquy avait lancé un sondage récemment sur `offerer_factories` vs `offerers_factories`, suivi d'une PR, donc restons sur :
```suggestion
from pcapi.core.offerers import factories as offerers_factories
```